plugins/Transitions/API.php

```diff
@@ -212,11 +218,12 @@ private function addInternalReferrers($logAggregator, &$report, $idaction, $acti
      * @param $report
      * @param $idaction
      * @param string $actionType
-     * @param $limitBeforeGrouping
+     * @param int $limitBeforeGrouping
      * @param boolean $includeLoops
      */
-    private function addFollowingActions($logAggregator, &$report, $idaction, $actionType, $limitBeforeGrouping, $includeLoops = false)
+    private function addFollowingActions($logAggregator, &$report, $idaction, $actionType, $limitBeforeGrouping = 0, $includeLoops = 0)
```

Comment:
this one should be reverted. There is no need to introduce a default for `$limitBeforeGrouping` and `$includeLoops` is meant to be a boolean